### DISCUSSION TITLE
请问一下，官方给出的稳定对应的是master分支还是v3.3.6

### DISCUSSION THREAD

**Discussion by weiwensi** [Netflix/conductor#2597] [Q&A] [answered]
请问一下，官方给出的稳定对应的是master分支还是v3.3.6

**Comment by apanicker-nflx** [comment, marked as answer]
Used google translate to get to this - 
> Excuse me, the official stable corresponds to the master branch or v3.3.6

The official stable version is the newest released version which is v3.3.6
